Model/XMLWriter.php

```diff
@@ -31,14 +29,19 @@ public function writeElement($name, $content = null, $cdata = false)
         }
     }
 
+    /**
+     * @param null $content
+     * @param bool $cdata
+     *
+     * @return bool|void
+     */
     public function writeRaw($content = null, $cdata = false)
     {
-	    $content = trim($content);
+        $content = trim($content);
```

Comment:
should you cast $content to a string here?  If not, why above?

Reply:
Yes. Fixed.